websurvey/src/routes/Form.svelte

```diff
@@ -80,12 +80,22 @@
         { type: "opinion_scale", component: Rating },
         { type: "email", component: ShortText },
     ];
+
+    const enter = 13;
+
+    function handleKeydown(event) {
```

Comment:
Why is this necessary? Shouldn't the forms all be forms and this funcitonality happens already?